### PR TITLE
Add package Microsoft.Extensions.Logging.Debug

### PR DESCRIPTION
**Pull request by aubergemediale**
...to fix build in VS 2017

### CODE REVIEW

**Comment by aubergemediale** [comment]
but this is credit to @derrick

**Comment by derrick** [comment]
I think you got the wrong Derrick.  I'm afraid I can't take credit for this.

**Comment by dangershony** [comment]
This is the right @Derrick- 
@aubergemediale can you get the branch up to date?

**Comment by Derrick-** [comment]
I think this can be closed, dupe of committed https://github.com/stratisproject/StratisBitcoinFullNode/pull/110

**Comment by dangershony** [comment]
@Derrick- I didn't understand which PR do you want to close?

**Comment by Derrick-** [comment]
This PR is dupe of #110 which is already committed, so this one can be closed.

**Comment by dangershony** [comment]
I don't think 110 is committed

**Comment by Derrick-** [comment]
Ahh yep, I thought it was committed so I rebased. My commit is gone from the PR.

But the change snuck in as part of my next commit:

https://github.com/stratisproject/StratisBitcoinFullNode/commit/65130cae397abd6f173b558be072b100bb06962d#diff-0330c405dd0d342eccd5457ff1bde5fa

So, needs closing

**Comment by aubergemediale** [comment]
it's up-to-date now!

File: Stratis.Bitcoin/Stratis.Bitcoin.csproj [hunk 1]
```diff
@@ -18,7 +18,11 @@
   </PropertyGroup>
 
   <ItemGroup>
```
Comment (dangershony) [inline Stratis.Bitcoin/Stratis.Bitcoin.csproj]: Ah new to git ;)
You are suppose to select whats inside the git signals
either whats inside HEAD or whats in Up Stream

This
```
+<<<<<<< HEAD
 +    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
 +=======
      <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.0.0" />
 +>>>>>>> upstream/master
```
Becomes
```
<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
```

Comment (dangershony) [inline Stratis.Bitcoin/Stratis.Bitcoin.csproj]: There is a learning curve with git but it becomes really awesome when you know how to use it

**Comment by aubergemediale** [comment]
it's fixed anyway, only version could be upgraded to 1.1.1

**Comment by dangershony** [comment]
So this is now fixed?

**Comment by aubergemediale** [comment]
yes, I tested it, it was a build and it works now, somebody else fiixed it, too by adding:
<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.0.0" />
My original pull request had:
<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />

But apparently v1.0.0 is also fine!

**Pull request closed without merging**